lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -204,7 +204,8 @@ sub login {
     _init_db($request);
     sanity_checks($database);
     $request->{login_name} = $version_info->{username};
-    if ($version_info->{status} eq 'does not exist'){
+    # $version_info->{status} isn't always defined
```

Comment:
Yes. The routine that sets it only do so on one branch of a if test. Logic remains ok, but lots of undefined messages are generated.